java/src/main/java/com/marginallyclever/makelangelo/MachineConfiguration.java

```diff
@@ -504,12 +512,12 @@ public void SaveConfig() {
 	/** 		TODO finish these methods.
 
 	 public boolean GetCanUseCloud() {
-		return prefs.getBoolean("can_use_cloud", false);
+		return machinePreferences.getBoolean("can_use_cloud", false);
 	}
```

Comment:
See my other [note](https://github.com/MarginallyClever/Makelangelo/pull/101/files#r30767803) as it is exactly the same.